ui/src/pages/Twitch/TwitchProfile/twitchProfile.scss

```diff
@@ -1,15 +1,15 @@
+.icon {
+  height: 85px;
+  width: 85px;
+  border-radius: 50%;
+  margin: 50px 20px 15px 20px;
+}
 .twitch {
-  &__profile{
-    margin-left: 25px;
-    &__icon {
-      &__true, &__false {
-        height: 85px;
-        width: 85px;
-        border-radius: 50%;
-      }
-      &__true {
-        margin: 100px;
-        display: block;
+  &__profile {
+    &__online {
+      width: 375px;
+      &__icon {
+        @extend .icon;
```

Comment:
Center the online view in its container like the offline state

Reply:
fixed it